base/src/main/java/com/austindroids/austinfeedsme/eventsfilter/EventFilterPresenter.kt

```diff
@@ -0,0 +1,51 @@
+package com.austindroids.austinfeedsme.eventsfilter
+
+import com.austindroids.austinfeedsme.data.Event
+import com.austindroids.austinfeedsme.data.EventsRepository
+import com.austindroids.austinfeedsme.data.FilterableEventsRepository
+import com.austindroids.austinfeedsme.di.modules.DataModule.Eventbrite
+import com.austindroids.austinfeedsme.di.modules.DataModule.Meetup
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.functions.Function3
+import io.reactivex.schedulers.Schedulers
+import java.util.*
+import javax.inject.Inject
+
+/**
+ * Created by darrankelinske on 8/4/16.
+ */
+class EventFilterPresenter @Inject
+constructor(private val filterableEventsRepository: FilterableEventsRepository, @param:Eventbrite private val eventbriteRepository: EventsRepository,
+            @param:Meetup private val meetupRepository: EventsRepository, private val view: EventFilterContract.View) : EventFilterContract.Presenter {
+    private var disposable: Disposable? = null
```

Comment:
Use `Disposables.disposed()` as a default, then you can make it non-nullable

Reply:
nice!